openedx/features/cms/helpers.py

```diff
@@ -3,13 +3,16 @@
 
 from dateutil.parser import parse
 from courseware.courses import get_course_by_id
+from opaque_keys.edx.keys import CourseKey
```

Comment:
Let's create a new app inside feature/cms directory and move all this code there

Reply:
This change will be done in the consequent PR for multiple reruns

Reply:
ok